### DISCUSSION TITLE
Add the ability for `rpnt`/`rmnt` type scriptlets to intercept text inside `data:` urls

### DISCUSSION THREAD

**Discussion by uBlock-user** [uBlockOrigin/uBlock-issues#3162] [General] [answered]
Case - `https://daotranslate.us/series/i-obtained-a-mythic-item/`

Anti-adblock script is placed inside a `data:` url which `rmnt`/`rpnt` type scriptlets can't intercept.

**Comment by gorhill** [comment, marked as answer]
`rmnt`/`rpnt` are specifically made to work on text nodes at DOM creation time, anything beyond that is incompatible with the code implementing these scriptlets -- these scriptlets are designed to do their job as fast as possible at DOM creation time through DOM mutation events, and trying to make them do more than this would result in an unmaintainable mess of code paths with difficult to predict behavior.

Before thinking of new scriptlets, let's try solution using currently supported syntax. How about:

    ||daotranslate.us^$csp=script-src 'self' *

**Reply by uBlock-user** [reply]
I already added `##+js(acs, ....)` scriptlet as a solution, so I thought you could do the same for these scriptlets. Anyways, closing.

**Reply by gorhill** [reply]
Also:

    daotranslate.us##.mdpDeblocker-wrapper
    daotranslate.us##.mdpDeblocker-blackout
    daotranslate.us##*:style(filter: none !important)

**Reply by uBlock-user** [reply]
Thanks, but already fixed with `acs` scriptlet.
